doc/cookbook/source/examples/classifier/multiclass_logisticregression.rst

```diff
@@ -0,0 +1,49 @@
+===============================
+Multi-class Logistic Regression
+===============================
+
+Multinomial logistic regression assigns the sample :math:`\mathbf{x}_i` to class :math:`c`
+based on :math:`P`, the probability for sample :math:`\mathbf{x}_i`: to be in class :math:`c`:
+
+.. math::
+
+    P(Y_i = c | \mathbf{X}_i) = \frac{\mathrm{exp}(\mathbf{\theta}^T_c\mathbf{X}_i)}{1+ \sum_{k=1}^{K}\mathrm{exp}(\mathbf{\theta}^T_k\mathbf{X}_i)}
+
+in which K is the number of classes.
+
+The loss function that needs to be minimized is:
+
+.. math::
+
+    {\min_{\mathbf{\theta}}}\sum_{k=1}^{K}\sum_{i=1}^{m}w_{ik}log(1+mathrm{exp}(-y_{ik}(\mathbf{x}_l^T\mathbf{a}_{ik} + c_k))) + \lambda\left \| \mathbf{x} \right \|_{l_1/l_q}
```

Comment:
You have that l_1/l_q from the manual I guess?
Do they say more about that? If so, also mention it below